lib/apiservers/engine/backends/portlayer/container_proxy.go

```diff
@@ -259,8 +269,29 @@ func (c *ContainerProxy) AddVolumesToContainer(handle string, config types.Conta
 	return handle, nil
 }
 
-// CommitContainerHandle() commits any changes to container handle.
+// AddPortsToContainer adds ports to a container, referenced by handle.
+// If an error is return, the returned handle should not be used.
 //
+// returns:
+//	modified handle
+func (c *ContainerProxy) AddPortsToContainer(handle string, config types.ContainerCreateConfig) (string, error) {
```

Comment:
What's the logic on the naming here? I had read this to now thinking it was adding exposed ports for some refactoring reason. Only now is it evident it's adding the interaction mechanism - this name is going to make less sense when it's a VMCI connection and not a serial port.